diff --git a/src/main/utils.ts b/src/main/utils.ts
index 4e7ec7e..69ee5e0 100644
--- a/src/main/utils.ts
+++ b/src/main/utils.ts
@@ -19,9 +19,9 @@ export function makeAsyncResource<T>(thing: T, dispose: () => Promise<void>): T
   const it = thing as T & AsyncDisposable;
 
   // eslint-disable-next-line no-restricted-syntax
   if (it[Symbol.asyncDispose]) {
-    throw new Error('Symbol.asyncDispose already exists');
+    return it;
   }
 
   // eslint-disable-next-line no-restricted-syntax
   it[Symbol.asyncDispose] = dispose;
diff --git a/dist/main.mjs b/dist/main.mjs
index 44b60ae..cc89d33 100644
--- a/dist/main.mjs
+++ b/dist/main.mjs
@@ -445,8 +445,8 @@ function He(r) {
 }
 function We(r, e) {
   const n = r;
   if (n[Symbol.asyncDispose])
-    throw new Error("Symbol.asyncDispose already exists");
+    return n;
   return n[Symbol.asyncDispose] = e, n;
 }
 function qe(r) {
